test/specs/android-specs/android_native_demo_test.js

```diff
@@ -2,25 +2,25 @@ describe('LocalAndroidTest', () => {
     it('quickStartAndroidNativeDemo', myTest)
  })
  async function myTest() {
-   const username = await $('id=com.experitest.ExperiBank:id/usernameTextField');
+   const username = await $('id=com.digitalai.ExperiBank:id/usernameTextField');
```

Comment:
again, "digitalai:", not just "digitalai"
all the changes to this file should be reverted